app/src/main/java/org/wikipedia/main/MainFragment.kt

```diff
@@ -270,15 +270,15 @@ class MainFragment : Fragment(), BackPressedHandler, FeedFragment.Callback, Hist
         }
     }
 
-    override fun onFeedMovePageToList(sourceReadingListId: Long, entry: HistoryEntry) {
+    override fun onFeedMovePageToList(sourceReadingList: Long, entry: HistoryEntry) {
         bottomSheetPresenter.show(childFragmentManager,
-                MoveToReadingListDialog.newInstance(sourceReadingListId, entry.title, InvokeSource.FEED))
+                MoveToReadingListDialog.newInstance(sourceReadingList, entry.title, InvokeSource.FEED))
     }
```

Comment:
It would be better if it is renamed on the `interface` side, which is following:

```
fun onFeedMovePageToList(sourceReadingListId: Long, entry: HistoryEntry)
```